src/test/java/com/microsoft/aad/msal4j/AuthorizationRequestUrlParametersTest.java

```diff
@@ -121,5 +123,6 @@ public void testBuilder_optionalParameters() throws UnsupportedEncodingException
         Assert.assertEquals(queryParameters.get("correlation_id"), "corr_id");
         Assert.assertEquals(queryParameters.get("login_hint"), "hint");
         Assert.assertEquals(queryParameters.get("domain_hint"), "domain_hint");
+        Assert.assertEquals(queryParameters.get("claims"), "{\"userinfo\":{\"given_name\":{\"essential\":true},\"nickname\":null,\"email\":{\"essential\":true},\"email_verified\":{\"essential\":true},\"picture\":null,\"http://example.info/claims/groups\":null},\"access_token\":{\"xms_cc\":{\"values\":[]}}}");
```

Comment:
These were just copied over from the integration test, where having values in that field would cause the server to return an error (since we don't know the valid capabilities values, or even if there are any valid ones). This test now takes in a set of two values, to properly testing merging of the capabilities set and the merging of capabilities into claims.